pkgs/by-name/ra/rainbowcrack/package.nix

```diff
@@ -0,0 +1,49 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  alglib,
+  unzip,
+}:
+
+stdenv.mkDerivation rec {
```

Comment:
```suggestion
stdenvNoCC.mkDerivation rec {
```

Reply:
stdenv and buildInputs = [ stdenv.cc.cc.lib ] worked fine earlier